configuration/rules-dsl.md

```diff
@@ -232,28 +231,28 @@ Contrary, the Actions `sendCommand(MyItem, "<new_state>")` and `postUpdate(MyIte
 
 The reasons lie within Java, the object-oriented programming language on which openHAB is built.
 Java and the Rules DSL have two basic types, primitives and Objects.
-A lower case letter data type after a `var` or a `val` statement, for example `var int`, indicates a primitive type. 
-An upper case letter data type after a `val` and `var` statement, for example `var Number` indicates an Object. 
+A lower case letter data type after a `var` or a `val` statement, for example `var int`, indicates a primitive type.
+An upper case letter data type after a `val` and `var` statement, for example `var Number` indicates an Object.
 Objects are more complex than primitives.
 
-Objects have methods associated that among others can make many necessary type conversions. 
-Using `Myitem.sendCommand(new_state)` or `Myitem.postUpdate(new_state)` can in most cases convert `new_state` into a type that Object `myItem` can apply. 
+Objects have methods associated that among others can make many necessary type conversions.
+Using `Myitem.sendCommand(new_state)` or `Myitem.postUpdate(new_state)` can in most cases convert `new_state` into a type that Object `myItem` can apply.
```

Comment:
Consider "will, in most cases, convert...".  Using "will" instead of "can" avoids having the same word twice in the same sentence.  Adding the commas increases readability.